anaconda3/alpine/Dockerfile

```diff
@@ -49,8 +49,8 @@ RUN ALPINE_GLIBC_BASE_URL="https://github.com/sgerrand/alpine-pkg-glibc/releases
 FROM alpine-glibc
 
 LABEL maintainer="Anaconda, Inc"
```

Comment:
Do we want to update to `alpine:3.15 as alpine-glibc`, see beginning of this file. The miniconda images are at 3.14.